db/payment_icons.yml

```diff
@@ -2210,3 +2210,7 @@
   name: qliro
   label: Qliro
   group: other
+-
+  name: cembrapay
```

Comment:
Any reason why CembraPay is not using any capitalization?

Reply:
I have updated it.